eslint.config.js

```diff
@@ -1,17 +1,17 @@
-"use strict";
+import globals from "globals";
+import eslintConfigESLint from "eslint-config-eslint";
+import eslintConfigESLintFormatting from "eslint-config-eslint/formatting";
+import markdown from "./src/index.js";
 
-module.exports = [
-    ...require("eslint-config-eslint/cjs").map(config => ({
-        ...config,
-        files: ["**/*.js"]
-    })),
+export default [
+    ...eslintConfigESLint,
     {
-        ...require("eslint-config-eslint/formatting"),
+        ...eslintConfigESLintFormatting,
         files: ["**/*.js"]
     },
```

Comment:
This wouldn't apply to .cjs files we now have. I'm not sure why these configs had `files`, but we can change this to be just:

```diff
- {
-     ...eslintConfigESLintFormatting,
-     files: ["**/*.js"]
- },
+ eslintConfigESLintFormatting,
```

(can't leave suggestion because of deleted line)